test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala

```diff
@@ -318,23 +318,160 @@ class MultipleDisposalsTriageControllerSpec
       "display form error when user enters numberOfProperties value <= 0" in {
 
         def test(data: (String, String)*)(expectedErrorMessageKey: String) =
-          testFormError(data: _*)(-5)(expectedErrorMessageKey)("multiple-disposals.howManyProperties.title")(
+          testFormError(data: _*)(-5)(expectedErrorMessageKey)("multipleDisposalsNumberOfProperties.title")(
             performAction
           )
 
-        test("numberOfProperties" -> "-5")("numberOfProperties.error.tooSmall")
+        test("multipleDisposalsNumberOfProperties" -> "-5")("multipleDisposalsNumberOfProperties.error.tooSmall")
 
       }
 
       "display form error when user enters numberOfProperties value > 999" in {
 
         def test(data: (String, String)*)(expectedErrorMessageKey: String) =
-          testFormError(data: _*)(1000)(expectedErrorMessageKey)("multiple-disposals.howManyProperties.title")(
+          testFormError(data: _*)(1000)(expectedErrorMessageKey)("multipleDisposalsNumberOfProperties.title")(
             performAction
           )
 
-        test("numberOfProperties" -> "1000")("numberOfProperties.error.tooLong")
+        test("multipleDisposalsNumberOfProperties" -> "1000")("multipleDisposalsNumberOfProperties.error.tooLong")
+
+      }
+
+    }
+
+    "handling requests to display the were uk resident page" must {
+
+      def performAction(): Future[Result] =
+        controller.wereYouAUKResident()(FakeRequest())
+
+      behave like redirectToStartWhenInvalidJourney(performAction, isValidJourney)
+
+      "display the page" in {
+        mockAuthWithNoRetrievals()
+        mockGetSession(sessionDataWithStartingNewDraftReturn(IncompleteMultipleDisposalsAnswers.empty)._1)
+
+        checkPageIsDisplayed(
+          performAction,
+          messageFromMessageKey("multipleDisposalsWereYouAUKResident.title"), { doc =>
+            doc.select("#back").attr("href") shouldBe triage.routes.MultipleDisposalsTriageController
+              .howManyDisposals()
+              .url
+            doc
+              .select("#content > article > form")
+              .attr("action") shouldBe routes.MultipleDisposalsTriageController
+              .wereYouAUKResidentSubmit()
+              .url
+          }
+        )
+      }
+
+    }
+
+    "handling submits on the were uk resident page" must {
+
+      def performAction(data: (String, String)*): Future[Result] =
+        controller.wereYouAUKResidentSubmit()(FakeRequest().withFormUrlEncodedBody(data: _*))
+
+      "user has not answered were uk resident section and " +
+        "redirect to dummy page when user selects yes" in {
```

Comment:
it's redirecting to check your answers. not a dummy page

Reply:
also, I'd try to make the tests sound like propert sentences. 

`The controller when handling submits on the were uk resident page must user has not answered ...`

doesn't quite make sense